.github/workflows/daily.yml

```diff
@@ -31,6 +33,7 @@ jobs:
           zip -r bevy_mobile_example.zip Payload
           mv bevy_mobile_example.zip bevy_mobile_example.ipa
 
+      # Browser Stack tests our code for us.
```

Comment:
not sure that comment is helping or even right

Reply:
Browser Stack appears to be be a testing service for multiple devices and browsers, though I do admit I'm not quite sure what specifically Bevy is using it for.

Could you explain what this workflow is doing, so I can correct the comment?

Reply:
Bevy uses Browser Stack to have access to real mobile devices. The part you documented just uploads the app built in Bevy CI to Browser Stack so that it's available for the next step when we run the test from Bevy CI

Reply:
I believe I fixed this in 9b89c9137ad42af9b7593dc72bfcc7e66840f246, thanks for explaining!